src/publicHolidayReminder.ts

```diff
@@ -1,5 +1,12 @@
-const { DateTime } = require('luxon');
-const ical = require('node-ical');
+import {
+  CalendarComponent,
+  CalendarResponse,
+  DateWithTimeZone,
+  VEvent,
+  async
```

Comment:
I think it would be best to alias this to avoid confusing it with the `async` keyword:

```suggestion
  async as ical
```